lms/static/scripts/frontend_apps/components/dashboard/DashboardActivityFilters.tsx

```diff
@@ -60,12 +60,13 @@ export default function DashboardActivityFilters({
   );
 
   return (
-    <div className="flex gap-2 md:w-1/2">
+    <div className="flex gap-2 flex-wrap">
```

Comment:
This is not a great, but for small screen resolutions it's better than the alternative.

With this, selects will be wrapping in new lines when there's not enough space, while otherwise, they generate horizontal scroll.

There's still improvements to be applied for a proper mobile-first layout.